From: Philipp Reisner <philipp.reisner@linbit.com>
Date: Wed, 30 Aug 2017 13:47:11 +0200
Subject: [PATCH] drbd: Fix allyesconfig build, fix recent commit
Git-commit: 5fc1efd5de1a1685e68d80981c7676c4d323d93c
Patch-mainline: v4.14-rc1
References: bsc#1104967,FATE#325924

Globals where prefixed with drbd_, that was missed in the
in #ifdef'nd code when it is built-in.

Signed-off-by: Philipp Reisner <philipp.reisner@linbit.com>
Fixes: 183ece30053f ("drbd: move global variables to drbd namespace and make some static")
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Acked-by: Hannes Reinecke <hare@suse.com>
---
 drivers/block/drbd/drbd_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/block/drbd/drbd_main.c b/drivers/block/drbd/drbd_main.c
index 56a5436d3f4f..92ba4c08835c 100644
--- a/drivers/block/drbd/drbd_main.c
+++ b/drivers/block/drbd/drbd_main.c
@@ -2989,7 +2989,7 @@ static int __init drbd_init(void)
 #ifdef MODULE
 		return -EINVAL;
 #else
-		minor_count = DRBD_MINOR_COUNT_DEF;
+		drbd_minor_count = DRBD_MINOR_COUNT_DEF;
 #endif
 	}
 
-- 
2.16.4

